action.yaml

```diff
@@ -193,11 +194,10 @@ runs:
         INPUT_UPLOAD_SERIES=${{ inputs.upload-series }}
         INPUT_UPLOAD_ID=
         EOF
-
         fi
 
     - name: Checkout
-      if: env.INPUT_TARGET_CHECKOUT
+      if: env.INPUT_TARGET_CHECKOUT && TEST_GITHUB_EVENT_PATH == ""
```

Comment:
@sxlijin is this what u mean by omitting this step? still seems to fail

Reply:
just don't specify `targetCheckout` in your payload

Reply:
hm i think im still struggling. -do u mind taking a look? sorry its kinda messy but just within action_tests.yml